### PR TITLE
Update DG & PPP

### PR DESCRIPTION
**Pull request by D-Limiter**

### CODE REVIEW

**Comment by A1WAYSD** [comment]
Please resolve the conflicts ASAP and can check group messages in Telegram. Thank you!

File: src/main/java/seedu/flashlingo/model/flashcard/FlashCard.java [hunk 1]
```diff
@@ -58,10 +58,11 @@ public FlashCard(OriginalWord originalWord, TranslatedWord translatedWord, Date
         this.originalWord = originalWord;
         this.isRemembered = isRemembered;
     }
+    //@@author D-Limiter
     public OriginalWord getOriginalWord() {
         return originalWord;
     }
-
+    //@@author D-Limiter
```
Comment (itsNatTan) [inline src/main/java/seedu/flashlingo/model/flashcard/FlashCard.java:65]: Didn't close authorship tag.

File: docs/images/StatsChart.png
Comment (itsNatTan) [inline docs/images/StatsChart.png:1]: Nice chart. If only it could have been added...

**Review by itsNatTan — CHANGES_REQUESTED** [review]
Close authorship tag

File: src/main/java/seedu/flashlingo/model/flashcard/FlashCard.java (hunk 1 above)
Comment (D-Limiter) [reply src/main/java/seedu/flashlingo/model/flashcard/FlashCard.java:65]: Hv corrected this

**Comment by D-Limiter** [comment]
> Please resolve the conflicts ASAP and can check group messages in Telegram. Thank you!

Hv resolved the conflicts @A1WAYSD

**Review by A1WAYSD — APPROVED** [review]
LGTM!

**Review by itsNatTan — APPROVED** [review]

**Pull request merged**